apps/server/test/support/cursor_support.ex

```diff
@@ -51,4 +51,12 @@ defmodule Lexical.Test.CursorSupport do
     end)
     |> IO.iodata_to_binary()
   end
+
+  defp starting_line do
+    1
+  end
+
+  defp starting_column do
+    1
+  end
```

Comment:
clear naming 👍

Reply:
I would not dislike it at all if this approach was taken for most `1`s and `0`s across the codebase